src/views/Notifications/Home/_components/_utils.tsx

```diff
@@ -0,0 +1,211 @@
+      </>
+    );
+  }
+  if (isDailyDigest(data)) {
+    return (
+      <>
+        <Trans i18nKey="notifications:index.activityType.dailyDigestComplete.cta" />
+        {isExpanded ? (
+          <Icon.ExpandLess className="margin-left-1" aria-hidden />
+        ) : (
+          <Icon.ExpandMore className="margin-left-1" aria-hidden />
+        )}
+      </>
+    );
+  }
+  return <></>;
+};
+export const TranslateStatusChange = ({
+  status
+}: {
+  status: string | null;
+}) => {
+  switch (status) {
+    case 'PLAN_DRAFT':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.plan_draft" />
+      );
+    case 'PLAN_COMPLETE':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.plan_complete" />
+      );
+    case 'ICIP_COMPLETE':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.icip_complete" />
+      );
+    case 'INTERNAL_CMMI_CLEARANCE':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.internal_cmmi_clearance" />
+      );
+    case 'CMS_CLEARANCE':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.cms_clearance" />
+      );
+    case 'HHS_CLEARANCE':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.hhs_clearance" />
+      );
+    case 'OMB_ASRF_CLEARANCE':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.omb_asrf_clearance" />
+      );
+    case 'CLEARED':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.cleared" />
+      );
+    case 'ANNOUNCED':
+      return (
+        <Trans i18nKey="notifications:index.dailyDigest.statusChange.announced" />
+      );
+    default:
+      return <></>;
+  }
+};
+
+export const translatePlanSections = (section: string) => {
+  switch (section) {
+    case 'plan_basics':
+      return i18next.t(
+        'notifications:index.dailyDigest.planSections.plan_basics'
+      );
+    case 'plan_payments':
+      return i18next.t(
+        'notifications:index.dailyDigest.planSections.plan_payments'
+      );
+    case 'plan_ops_eval_and_learning':
+      return i18next.t(
+        'notifications:index.dailyDigest.planSections.plan_ops_eval_and_learning'
+      );
+    case 'plan_participants_and_providers':
+      return i18next.t(
+        'notifications:index.dailyDigest.planSections.plan_participants_and_providers'
+      );
+    case 'plan_beneficiaries':
+      return i18next.t(
+        'notifications:index.dailyDigest.planSections.plan_beneficiaries'
+      );
+    case 'plan_general_characteristics':
+      return i18next.t(
+        'notifications:index.dailyDigest.planSections.plan_general_characteristics'
+      );
+    default:
+      return '';
+  }
+};
+
+export const pushValuesToChangesArray = (
+  obj: ChangeTypes,
+  changesArray: string[]
```

Comment:
I think this function should only take in that first parameter.  `changesArray` does not need to be instantiated outside this function and then passed in, rather this function should return an array as a part of the instantiation.

Ex: 

`const changesArray: string[] = pushValuesToChangesArray(changes);`